src/components/deleteButton.vue

```diff
@@ -1,22 +1,26 @@
 <template>
-  <img
-    :src="trashIcon"
-    @click="$emit('click')"
+  <crown-button
     id="delete-button"
-    data-test="delete-button"
-    aria-hidden="true"
-    data-prefix="fas"
-    data-icon="trash-alt"
-    class="svg-inline--fa fa-trash-alt fa-w-14"
-    role="img"
-    alt="Delete this node"
+    aria-label="Delete this node"
+    @click="$emit('click')"
```

Comment:
Since this click listener is also just being passed down, I think you can replace it with `v-on="$listeners"`; `crown-button` is really a transparent wrapper.